frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -1,13 +1,17 @@
 import * as _ from 'lodash-es';
 import * as React from 'react';
 import * as fuzzy from 'fuzzysearch';
+// Work around eslint error WithTranslation not found in 'react-i18next'
+// eslint-disable-next-line
```

Comment:
Is this still needed? If you rebase, can you check if the import works now?

Reply:
After rebasing I'm still getting this error with out it.
```
/Users/stevegoodwin/Git/console/frontend/public/components/utils/storage-class-dropdown.tsx
  4:10  error  WithTranslation not found in 'react-i18next'  import/named
```